teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<ul><li>It just came to my mind here, but maybe introduce make a note when first used that 'target variable' should not be confused with (protein) target.</li><li>Is the MSE necessarily between 0 and 1?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Good point about target variable. NOTE added in the first mention. </p><p>Regarding MSE, indeed it has to be positive but not necessarily lower than 1. Same goes for RMSE. I fixed it. </p>